src/index.md

```diff
@@ -4,6 +4,13 @@
 [![GitHub contributors](https://img.shields.io/github/contributors/google/comprehensive-rust?style=flat-square)](https://github.com/google/comprehensive-rust/graphs/contributors)
 [![GitHub stars](https://img.shields.io/github/stars/google/comprehensive-rust?style=flat-square)](https://github.com/google/comprehensive-rust/stargazers)
 
+<LanguagePicker>
```

Comment:
Shows that it works inside of markdown too

Reply:
Click events don't work, it is because of mdbook that prevents bubbling in nested HTML. We might be able to fix this with z-index. I don't think it is a priority right now.

Reply:
We can click on code blocks, so there definitely is a way anyhow

Reply:
I guess we can take this out?